gcc/cp/cp-tree.h

```diff
@@ -3204,6 +3204,18 @@ struct GTY(()) lang_decl {
 #define CONTRACT_HELPER(NODE) \
  (LANG_DECL_FN_CHECK (NODE)->contract_helper)
 
+/* In VIEW_CONVERT_EXPR, set when this node is a const wrapper.  Used to
+   constify entities inside contract assertions.  */
+
+#define EXPR_CONTRACT_CONST_WRAPPER_P(NODE) \
```

Comment:
Just name this CONTRACT_CONSTIFY_EXPR_P? It's like a VIEW_CONVERT_EXPR, but with a more specific purpose.

Reply:
done